### ISSUE TITLE
FastPitch example fails to play audio using ffplay and browser

### ISSUE THREAD

**Issue by anagrath** [NVIDIA/DeepLearningExamples#1251]
I am trying to train a fastpitch.  Prior to training, I downloaded the pretrained models to run an example to hear the output.

I am using NGC via: 
bash scripts/docker/build.sh
bash scripts/docker/interactive.sh

I changed the Dockerfile to use PyTorch 22.12-py3

I ran this outside of the docker:
bash scripts/download_dataset.sh

I ran this inside of the docker:
bash scripts/prepare_dataset.sh

Documentation claims that download_dataset should bring down pretrained waveglow model, this does not appear to be true any longer.  To download the models I ran:
scripts/download_models.sh

Here is a ls of my pretrained directory:
-rw-r--r-- 1 root     root     164410 Jan 19 09:55 audio_0.wav
drwxrwxr-x 2   xxx xxxx  4096 Jan 19 09:04 fastpitch
drwxrwxr-x 2 xxx xxxx   4096 Jan 19 09:04 hifigan
drwxrwxr-x 2 xxx xxxx   4096 Jan 18 15:48 waveglow

each directory has a zip file and a pt file within.

I launched the docker using
bash scripts/docker/interactive.sh

Following the inference example in the documentation I ran as follows:

python inference.py --cuda --fastpitch pretrained_models/fastpitch/nvidia_fastpitch_210824.pt --waveglow pretrained_models/waveglow/nvidia_waveglow256pyt_fp16.pt --wn-channels 256 -i phrases/phrase_1_64.txt -o pretrained_models/ --amp

I removed  --energy-conditioning  because I could not find this option in the inference.py file (I could not understand what it does).
I added --amp due to how long this process takes.

The process ramps up the memory on my 1 4090 GPU to about 1500MB, I can see the CPU running at 100%.  I have never sat in front of the computer to see process completion (I do not know how long it takes, but it takes long enough to walk down the street to get a coffee and come back).  On one of the runs I could see that the CPU is at 100% and the CPU memory was going up by 3MB every few seconds.  On the one time I saw the process complete I noticed it had brought in about 1.5 gig (same as graphics card).  I have no idea why this is so slow.

I created a test.txt file that matches the documentation's recommendation because I was not sure if the tsv file in the example was breaking the model:
`<output wav file name>|<utterance>`

Here is my test.txt:
ined_models$ cat ../phrases/test.txt 
test1.wav|Can you hear the words that are coming out of my mouth?

Here is the output from the run of the inference.py from above:
root@66f7d9e1bb67:/workspace/fastpitch# python inference.py --cuda --fastpitch pretrained_models/fastpitch/nvidia_fastpitch_210824.pt --waveglow pretrained_models/waveglow/nvidia_waveglow256pyt_fp16.pt --wn-channels 256 -i phrases/phrase_1_64.txt -o pretrained_models/ --amp
Thread affinity: {0}
DLL 2023-01-19 16:49:51.846759 - PARAMETER | input :  phrases/phrase_1_64.txt
DLL 2023-01-19 16:49:51.846801 - PARAMETER | output :  pretrained_models/
DLL 2023-01-19 16:49:51.846827 - PARAMETER | log_file : 
DLL 2023-01-19 16:49:51.846852 - PARAMETER | save_mels :  False
DLL 2023-01-19 16:49:51.846877 - PARAMETER | cuda :  True
DLL 2023-01-19 16:49:51.846900 - PARAMETER | cudnn_benchmark :  False
DLL 2023-01-19 16:49:51.846923 - PARAMETER | l2_promote :  False
DLL 2023-01-19 16:49:51.846945 - PARAMETER | fastpitch :  pretrained_models/fastpitch/nvidia_fastpitch_210824.pt
DLL 2023-01-19 16:49:51.846969 - PARAMETER | waveglow :  pretrained_models/waveglow/nvidia_waveglow256pyt_fp16.pt
DLL 2023-01-19 16:49:51.846992 - PARAMETER | waveglow_sigma_infer :  0.9
DLL 2023-01-19 16:49:51.847015 - PARAMETER | hifigan : 
DLL 2023-01-19 16:49:51.847037 - PARAMETER | denoising_strength :  0.0
DLL 2023-01-19 16:49:51.847064 - PARAMETER | hop_length :  256
DLL 2023-01-19 16:49:51.847087 - PARAMETER | win_length :  1024
DLL 2023-01-19 16:49:51.847112 - PARAMETER | sampling_rate :  22050
DLL 2023-01-19 16:49:51.847134 - PARAMETER | max_wav_value :  32768.0
DLL 2023-01-19 16:49:51.847158 - PARAMETER | amp :  True
DLL 2023-01-19 16:49:51.847180 - PARAMETER | batch_size :  64
DLL 2023-01-19 16:49:51.847204 - PARAMETER | warmup_steps :  0
DLL 2023-01-19 16:49:51.847226 - PARAMETER | repeats :  1
DLL 2023-01-19 16:49:51.847248 - PARAMETER | torchscript :  False
DLL 2023-01-19 16:49:51.847275 - PARAMETER | checkpoint_format :  pyt
DLL 2023-01-19 16:49:51.847305 - PARAMETER | torch_tensorrt :  False
DLL 2023-01-19 16:49:51.847328 - PARAMETER | report_mel_loss :  False
DLL 2023-01-19 16:49:51.847349 - PARAMETER | ema :  False
DLL 2023-01-19 16:49:51.847371 - PARAMETER | dataset_path : 
DLL 2023-01-19 16:49:51.847393 - PARAMETER | speaker :  0
DLL 2023-01-19 16:49:51.847415 - PARAMETER | affinity :  single
DLL 2023-01-19 16:49:51.847437 - PARAMETER | fade_out :  10
DLL 2023-01-19 16:49:51.847463 - PARAMETER | pace :  1.0
DLL 2023-01-19 16:49:51.847486 - PARAMETER | pitch_transform_flatten :  False
DLL 2023-01-19 16:49:51.847534 - PARAMETER | pitch_transform_invert :  False
DLL 2023-01-19 16:49:51.847558 - PARAMETER | pitch_transform_amplify :  1.0
DLL 2023-01-19 16:49:51.847581 - PARAMETER | pitch_transform_shift :  0.0
DLL 2023-01-19 16:49:51.847604 - PARAMETER | pitch_transform_custom :  False
DLL 2023-01-19 16:49:51.847627 - PARAMETER | text_cleaners :  ['english_cleaners_v2']
DLL 2023-01-19 16:49:51.847653 - PARAMETER | symbol_set :  english_basic
DLL 2023-01-19 16:49:51.847676 - PARAMETER | p_arpabet :  0.0
DLL 2023-01-19 16:49:51.847700 - PARAMETER | heteronyms_path :  cmudict/heteronyms
DLL 2023-01-19 16:49:51.847722 - PARAMETER | cmudict_path :  cmudict/cmudict-0.7b
FastPitch: Loading pretrained_models/fastpitch/nvidia_fastpitch_210824.pt...
FastPitch: Found model config saved in the checkpoint.
/usr/local/lib/python3.8/dist-packages/torch/functional.py:1690: UserWarning: torch.lu is deprecated in favor of torch.linalg.lu_factor / torch.linalg.lu_factor_ex and will be removed in a future PyTorch release.
LU, pivots = torch.lu(A, compute_pivots)
should be replaced with
LU, pivots = torch.linalg.lu_factor(A, compute_pivots)
and
LU, pivots, info = torch.lu(A, compute_pivots, get_infos=True)
should be replaced with
LU, pivots, info = torch.linalg.lu_factor_ex(A, compute_pivots) (Triggered internally at /opt/pytorch/pytorch/aten/src/ATen/native/BatchLinearAlgebra.cpp:2018.)
  return torch._lu_with_info(A, pivot=pivot, check_errors=(not get_infos))
WaveGlow: Loading pretrained_models/waveglow/nvidia_waveglow256pyt_fp16.pt...
WaveGlow: Found model config saved in the checkpoint.
she sells seashells by the seashore, shells she sells are great
Warmup: 0it [00:00, ?it/s]
DLL 2023-01-19 16:55:07.997120 - 0 | fastpitch frames/s            1915.39 frames/s
DLL 2023-01-19 16:55:07.997228 - 0 | fastpitch latency             0.16759 s
DLL 2023-01-19 16:55:08.430886 - 0 | waveglow samples/s          189524.69 samples/s
DLL 2023-01-19 16:55:08.430989 - 0 | waveglow latency              0.43359 s
DLL 2023-01-19 16:55:08.432236 - 0 | latency                       0.60118 s
DLL 2023-01-19 16:55:08.432437 - () | avg fastpitch tokens/s         375.92 tokens/s
DLL 2023-01-19 16:55:08.432487 - () | avg fastpitch frames/s        1915.39 frames/s
DLL 2023-01-19 16:55:08.432528 - () | avg fastpitch latency         0.16759 s
DLL 2023-01-19 16:55:08.432561 - () | avg fastpitch RTF               22.24 x
DLL 2023-01-19 16:55:08.432588 - () | avg fastpitch RTF@64            22.24 x
DLL 2023-01-19 16:55:08.432878 - () | 90% fastpitch latency         0.16759 s
DLL 2023-01-19 16:55:08.433066 - () | 95% fastpitch latency         0.16759 s
DLL 2023-01-19 16:55:08.433242 - () | 99% fastpitch latency         0.16759 s
DLL 2023-01-19 16:55:08.433307 - () | avg waveglow samples/s      189524.69 samples/s
DLL 2023-01-19 16:55:08.433350 - () | avg waveglow latency          0.43359 s
DLL 2023-01-19 16:55:08.433380 - () | avg waveglow RTF                 8.60 x
DLL 2023-01-19 16:55:08.433408 - () | avg waveglow RTF@64              8.60 x
DLL 2023-01-19 16:55:08.433573 - () | 90% waveglow latency          0.43359 s
DLL 2023-01-19 16:55:08.433747 - () | 95% waveglow latency          0.43359 s
DLL 2023-01-19 16:55:08.433917 - () | 99% waveglow latency          0.43359 s
DLL 2023-01-19 16:55:08.433975 - () | avg samples/s               136691.16 samples/s
DLL 2023-01-19 16:55:08.434011 - () | avg letters/s                  104.79 letters/s
DLL 2023-01-19 16:55:08.434047 - () | avg latency                   0.60118 s
DLL 2023-01-19 16:55:08.434077 - () | avg RTF                          6.20 x
DLL 2023-01-19 16:55:08.434104 - () | avg RTF@64                       6.20 x
DLL 2023-01-19 16:55:08.434265 - () | 90% latency                   0.60118 s
DLL 2023-01-19 16:55:08.434434 - () | 95% latency                   0.60118 s
DLL 2023-01-19 16:55:08.434602 - () | 99% latency                   0.60118 s

Here is the output from ffplay of the resulting audio_0.wav:
(base) xxx@xxxx:~/Code/ml/DeepLearningExamples/PyTorch/SpeechSynthesis/FastPitch/pretrained_models$ ffplay audio_0.wav 
ffplay version 4.4.2-0ubuntu0.22.04.1 Copyright (c) 2003-2021 the FFmpeg developers
  built with gcc 11 (Ubuntu 11.2.0-19ubuntu1)
  configuration: --prefix=/usr --extra-version=0ubuntu0.22.04.1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-pocketsphinx --enable-librsvg --enable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
[wav @ 0x7f4f10000cc0] Could not find codec parameters for stream 0 (Audio: none ([3][0][0][0] / 0x0003), 22050 Hz, 1 channels, 352 kb/s): unknown codec
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
Input #0, wav, from 'audio_0.wav':
  Duration: 00:00:03.73, bitrate: 352 kb/s
  Stream #0:0: Audio: none ([3][0][0][0] / 0x0003), 22050 Hz, 1 channels, 352 kb/s
No decoder could be found for codec none
Failed to open file 'audio_0.wav' or configure filtergraph

---- Anyone know why the wav file is malformed or does not have a codec?
